adafruit_ble_file_transfer.py

```diff
@@ -75,38 +80,44 @@ def bind(self, service):
         bound_characteristic = super().bind(service)
         return _bleio.PacketBuffer(bound_characteristic, buffer_size=4)
 
+
 class FileTransferService(Service):
     """Simple (not necessarily fast) BLE file transfer service. It implements basic CRUD operations.
 
     The server dictates data transfer chunk sizes so it can minimize buffer sizes on its end.
     """
 
-    uuid = StandardUUID(0xfebb)
+    # pylint: disable=too-few-public-methods
+
+    uuid = StandardUUID(0xFEBB)
     version = Uint32Characteristic(uuid=FileTransferUUID(0x0100))
     raw = _TransferCharacteristic()
     # _raw gets shadowed for each MIDIService instance by a PacketBuffer. PyLint doesn't know this
     # so it complains about missing members.
     # pylint: disable=no-member
 
-    # Actions
+    # Commands
     INVALID = 0x00
     READ = 0x01
     WRITE = 0x02
     DELETE = 0x03
     MKDIR = 0x04
     LISTDIR = 0x05
```

Comment:
RENAME, COPY operations are not expected to be useful?  I'm imagining WRITE+RENAME to replace content "atomically" and leave a functioning device if something interferes, but maybe it's not a great idea.

EXEC? (of a Python code string)... I guess this list can be expanded in the future but we would be promising to implement anything on the list for awhile, so better to keep it small.  Forget I mentioned anything.

Reply:
I'm really trying to keep it simple now. We can add RENAME and/or COPY later if we find we need it. WRITE+RENAME is a pretty good idea that we should think about further.

I wouldn't do EXEC ever because I want this protocol to be agnostic of what is running the server. For example, it may be Arduino instead of CircuitPython.